media-libs/oidn/files/glibc-1.3.0.patch

```diff
@@ -0,0 +1,613 @@
+From 4180502f326dccd5e46ab67176b53cc800e32f7b Mon Sep 17 00:00:00 2001
```

Comment:
Missing link at the top to the relevant bug in Gentoo + upstream commit. See [here](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto).